app/templates/components/ui-table/cell/admin/users/cell-actions.hbs

```diff
@@ -2,12 +2,12 @@
   {{#ui-popup tagName='a' click=(action moveToUserDetails record.id) content=(t 'View') class='ui icon button'}}
     <i class="unhide icon"></i>
   {{/ui-popup}}
-  {{#unless record.isSuperAdmin}}
+  {{#if (and (eq record.deletedAt null) (not-eq record.isSuperAdmin true))}}
```

Comment:
Try not to keep complex conditionals in the template. The template should only have minimal logic. Move this to a computed property on the javascript file